.env.development

```diff
@@ -2,3 +2,4 @@ REACT_APP_DEV_MNEMONIC='benefit crew supreme gesture quantum web media hazard th
 REACT_APP_DEV_GALAXY_TICKET='~wacfus-dabpex-danted-mosfep-pasrud-lavmer-nodtex-taslus-pactyp-milpub-pildeg-fornev-ralmed-dinfeb-fopbyr-sanbet-sovmyl-dozsut-mogsyx-mapwyc-sorrup-ricnec-marnys-lignex'
 REACT_APP_ROPSTEN_KEY='b7d2af9f01534031ba773374f766ef65'
 REACT_APP_HD_PATH="m/44'/60'/0'/0/0"
+REACT_APP_STUB_LOCAL='false'
```

Comment:
This is good to have in here, but can we get unrelated changes as separate commits next time? Same goes for some minor cleanup you did in this PR. The more single-purpose a commit is, the better imo. (:

Reply:
yeah totally  — will move those changes to a new PR

Reply:
"Will do in the future" would've been fine too, but thanks. (;